sphinx_scality/footer.html

```diff
@@ -10,16 +10,7 @@
       <a href="#">Privacy Policy</a> |
       <a href="#">Training</a>
     </div>
-    <div class="copyright">
```

Comment:
No more copyrights? Why not? And what's the link with the commit message `Enhanced titles`?